radixdlt/src/main/java/com/radixdlt/consensus/BFTEventReducer.java

```diff
@@ -113,15 +113,14 @@ private void proceedToView(View nextView) {
 
 	private void processQC(QuorumCertificate qc) {
 		// commit any newly committable vertices
-		this.safetyRules.process(qc)
-			.ifPresent(vertexId -> {
-				final Vertex vertex = vertexStore.commitVertex(vertexId);
-				log.info("{}: Committed vertex: {}", this.getShortName(), vertex);
-				final Atom committedAtom = vertex.getAtom();
-				if (committedAtom != null) {
-					mempool.removeCommittedAtom(committedAtom.getAID());
-				}
-			});
+		this.safetyRules.process(qc).ifPresent(vertexId -> {
```

Comment:
Is the reformatting here intentional?

Reply:
Yes, just making it more consistent with other uses of ifPresent.

Reply:
can we remove this as it will make merging later more difficult as this is definitely changed in other branches.